x-pack/plugins/siem/cypress/tasks/create_new_case.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Case } from '../objects/case';
+
+import {
+  BACK_TO_CASES_BTN,
+  DESCRIPTION_INPUT,
+  SUBMIT_BTN,
+  INSERT_TIMELINE_BTN,
+  LOADING_SPINNER,
+  TAGS_INPUT,
+  TIMELINE,
+  TIMELINE_SEARCHBOX,
+  TITLE_INPUT,
+} from '../screens/create_new_case';
+
+export const backToCases = () => {
+  cy.get(BACK_TO_CASES_BTN).click({ force: true });
+};
+
+export const createNewCase = (newCase: Case) => {
+  cy.get(TITLE_INPUT).type(newCase.name, { force: true });
+  newCase.tags.forEach(tag => {
+    cy.get(TAGS_INPUT).type(`${tag}{enter}`, { force: true });
+  });
+  cy.get(DESCRIPTION_INPUT).type(`${newCase.description} `, { force: true });
+
+  cy.get(INSERT_TIMELINE_BTN).click({ force: true });
+  cy.get(TIMELINE_SEARCHBOX).type(`${newCase.timeline.title}{enter}`);
+  cy.get(TIMELINE).should('be.visible');
```

Comment:
this could happen before the filter is applied resulting in the insertion of an unrelated timeline
<img width="887" alt="Screen Shot 2020-04-20 at 12 31 01 PM" src="https://user-images.githubusercontent.com/6935300/79786655-09c0eb00-8303-11ea-9c1b-3c527f2cd42a.png">